nservicebus/hosting/nservicebus-host/logging-configuration.md

```diff
@@ -12,14 +12,14 @@ include: host-deprecated-warning
 
 ## Logging configuration
 
-This article explains how to customize default configuration. Refer to the [Profiles - Logging](/nservicebus/hosting/nservicebus-host/profiles.md#logging) section for more details about logging configuration in built-in profiles.
+This article explains how to customize the logging configuration when using the NServiceBus Host. For more details about logging configuration in built-in profiles, refer to the [NServiceBus.Host Profiles - Logging](profiles.md#logging) section.
 
-Logging configuration for NServiceBus Host can be customized by providing endpoint configuration or using profiles.
+## Constructor of IConfigureThisEndpoint implementation
```

Comment:
I'm a bit confused here, shouldn't that be in the partial? Don't we repeat the same information in `logging-configuration_customize_host_[5,].partial.md` later on?

I'd stick to what we have in partial, as there's an additional sippet that we don't have here. So I suggest to skip the section title and "It is advised to customize logging in the constructor of the class that implements `IConfigureThisEndpoint`. This is recommended as this class is the earliest opportunity to initialize [any custom logging framework](/nservicebus/logging/#custom-logging)"